src/docs/implementation/ArcCommanderManual/arc_access.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+title: "DataHUB access for ARC synchronization"
+published: 2022-12-20
+author: Martin Kuhl
+author_github: martin-kuhl
+add toc: false
+add support: false
```

Comment:
Why is the support removed from all pages. Is this on intention or should this be changed?

Reply:
The recurrent block honestly annoyed me. Especially for the very small articles / bricks this can take up much more than the actual content.

Reply:
Even if the other points of thi pull request are merged soon, I suggest to place this point as a separate issue and maybe also as a message in a shared channel for discussion with the teaching_material team to get the DataPLANT side's opinion after the break and adjust accordingly if required.

Reply:
Added as issue #62